charts/landscaper/templates/deployment-webhooks.yaml

```diff
@@ -28,8 +28,13 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       {{- if .Values.serviceAccount.create }}
+      {{- if not .Values.webhooksServer.landscaperKubeconfig }}
       serviceAccountName: {{ include "landscaper.webhooks.serviceAccountName" . }}
       {{- end }}
+      {{- end }}
+      {{- if .Values.webhooksServer.landscaperKubeconfig }}
```

Comment:
```suggestion
      {{- else }}
```

Reply:
I even think that checking for `.Values.serviceAccount.create` is not correct in this case.
We also do not check for this value in the controller deployment.
When `.Values.webhooksServer.landscaperKubeconfig` is not set, the deployment needs to have a serviceAccount bound with the correct roles attached, despite of  `.Values.serviceAccount.create`.